### DISCUSSION TITLE
【音乐自动暂停播放】自动播放开启后，打开多个页面、音乐重音；希望切换标签后自动暂停播放

### DISCUSSION THREAD

**Discussion by expoli** [tangly1024/NotionNext#2027] [Ideas]
**描述bug**
在打开播放器的自动播放时，打开多个页面、多个页面会同时播放导致重音

**复现步骤**
启用播放器、并开启自动播放

**期望的正常结果**
启用播放器、并开启自动播放，打开多个页面只有激活活动的浏览器标签页播放音乐

**截图**
相关的页面，应该用结果

**环境**

- 操作系统: [例如. iOS, Android, macOS, windows]
- 浏览器 [例如. chrome, safari, firefox]
- 版本 [e.g. 3.8.0]

**补充说明**
与问题相关的其它说明

**Comment by lifeafter619** [comment]
可能这个issue会有所帮助
https://github.com/DIYgod/APlayer/issues/725
